.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "webpack",
+  "rules": {
```

Comment:
I this added intentionally or was this the previous config ?  `webpack-defaults` doesn't always remove all previous settings, when applied for the first time, you need to double check here :) 

⚠️ This can also happen in the `.bablerc` file